internal/deployment/deployment_test.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deployment
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	nfdv1 "sigs.k8s.io/node-feature-discovery-operator/api/v1"
+	"sigs.k8s.io/yaml"
+)
+
+var _ = Describe("SetMasterDeploymentAsDesired", func() {
+	var (
+		deploymentAPI DeploymentAPI
+	)
+
+	BeforeEach(func() {
+		deploymentAPI = NewDeploymentAPI(scheme)
+	})
+
+	ctx := context.Background()
+
+	It("good", func() {
```

Comment:
nit: same here, could be a bit more descriptive

Reply:
Fixed